### ISSUE TITLE
My work item

### ISSUE THREAD

**Issue by jascam** [jascam/MigrationTest#1]
My work item on the workspace homepage does not show my work items

#### Migrated CodePlex Work Item Details
CodePlex Work Item ID: '162'
Vote count: '0'

**Comment by jascam** [comment]
My user name in the system is one thing (bsimser) and I use that to log on. When I look at the Source Code tab on my project, it says my user name is bsimser_cp. I've used that to connect via Team Explorer (in VS2005 Team Suite) and all was good. 

I went through and assigned work items to myself (I'm the only person on the project) and the user list only offered bsimser_cp as a choice (along with the Developers group). 

Now when I log on via the web interface, there are no Work Items shown as assigned to me however looking at the Issue Tracker in the web interface, it shows items assigned to "bsimser", not "bsimser_cp".

Something is amiss.

This comment was posted by  on 5/24/2006
